README.md

```diff
@@ -49,8 +49,8 @@ descriptions of all these features.
 ## Releases & Documentation
 
 
-**Our latest, stable release is `Release 13`. Click
-[here](https://github.com/Unity-Technologies/ml-agents/tree/release_13_docs/docs/Readme.md)
+**Our latest, stable release is `Release 14`. Click
+[here](https://github.com/Unity-Technologies/ml-agents/tree/release_14_docs/docs/Readme.md)
 to get started with the latest release of ML-Agents.**
 
 The table below lists all our releases, including our `master` branch which is
```

Comment:
I missed it, you might also want to update the table

Reply:
yeah, I'll do it in another PR